shared/xccdf/services/ssh.xml

```diff
@@ -360,6 +370,31 @@ recognition and reporting of unauthorized account use.
 <ref prodtype="rhel7" nist="AC-9" disa="366" ossrg="SRG-OS-000480-GPOS-00227" stigid="040360"/>
 </Rule>
 
+<Rule id="sshd_set_loglevel_info" severity="low" prodtype="debian8">
+<title>Set LogLevel to INFO</title>
+<description>The INFO parameter specifices that record login and logout activity will be logged.
+To specify the log level in
+SSH, add or correct the following line in the <tt>/etc/ssh/sshd_config</tt> file:
+<pre>LogLevel INFO</pre>
+</description>
+<ocil clause="it is commented out or is not enabled">
+To check if PrintLastLog is enabled or set correctly, run the
+following command:
+<pre>$ sudo grep "^LogLevel" /etc/ssh/sshd_config</pre>
+If configured properly, output should be <pre>LogLevel INFO</pre>
+</ocil>
+<rationale>
+SSH provides several logging levels with varying amounts of verbosity. <tt>DEBUG</tt> is specifically
+not recommended other than strictly for debugging SSH communications since it provides
+so much data that it is difficult to identify important security information. <tt>INFO</tt> level is the
+basic level that only records login activity of SSH users. In many situations, such as Incident
+Response, it is important to determine when a particular user was active on a system. The
+logout record can eliminate those users who disconnected, which helps narrow the field.
+</rationale>
+<oval id="sshd_set_loglevel_info" />
+<ref prodtype="debian8" cis="9.3.2"/>
```

Comment:
@redhatrises I'll be submitting a PR soon, which uses shared checks/guide for debian